### PR TITLE
Fix for issue180 - Disclaimer text

### LINKED ISSUE #180: Disclaimer - Fix capitalisation in content
Can we please verify that `anaesthetic`, `critical care` and `clinician` aren’t proper nouns and should be lower case?

Image from iOS (2)

Image from iOS (1)

### PR DESCRIPTION
**Pull request by lukesleeman**
Some minor changes to the disclaimer text.  Closes #180 

- [x] `release-notes.txt` has been updated.

## Changes

- Fix capitalisation
- Remove repeated "have"

## Screenshots

image

## Things to note

I've bumped up the disclaimer to V2 - should be a good test to ensure everybody needs to re-accept it.

### CODE REVIEW

File: lib/strings.dart [hunk 1]
```diff
@@ -11,16 +11,16 @@ class Strings {
   // *** WARNING ***
   // When disclaimerBody changes in any way, increment disclaimerVersion by one
   static const String disclaimerBody = '''
-This app is designed to be used by Anaesthetic and Critical Care doctors at Western Health who are caring for patients during the COVID-19 Pandemic. The guidelines, protocols and advice within this app are designed to educate staff and suggest recommended management of patients with COVID-19. The recommendations within this application do not indicate an exclusive course of action as the results must be contextualised to the patient at hand and variations may be appropriate.
+This app is designed to be used by anaesthetic and critical care doctors at Western Health who are caring for patients during the COVID-19 Pandemic. The guidelines, protocols and advice within this app are designed to educate staff and suggest recommended management of patients with COVID-19. The recommendations within this application do not indicate an exclusive course of action as the results must be contextualised to the patient at hand and variations may be appropriate.
```
Comment (kimngyn) [inline lib/strings.dart]:
```suggestion
This app is designed to be used by the anaesthetist and critical care doctors at Western Health who are caring for patients during the COVID-19 pandemic. The guidelines, protocols and advice within this app are designed to educate staff and suggest recommended management of patients with COVID-19. The recommendations within this application do not indicate an exclusive course of action as the results must be contextualised to the patient at hand and variations may be appropriate.
```

File: lib/view/disclaimer_view.dart [hunk 2]
```diff
@@ -77,7 +77,7 @@ class _DisclaimerViewState extends State<DisclaimerView> {
                           ),
                         ),
                         Text(
-                            'This app is for Clinician and hospital staff use ONLY\n',
+                            'This app is for clinician and hospital staff use ONLY\n',
```
Comment (kimngyn) [inline lib/view/disclaimer_view.dart:80]: Good!

**Review by kimngyn — CHANGES_REQUESTED** [review]

File: lib/view/disclaimer_view.dart (hunk 2 above)
Comment (kimngyn) [inline lib/view/disclaimer_view.dart]:
```suggestion
                            'This app is for clinicians and hospital staff use ONLY\n',
```

**Review by workerbee22 — APPROVED** [review]
LGTM

**Comment by workerbee22** [comment]
@luke good idea taking to version 2.

**Comment by workerbee22** [comment]
@kimngyn I also spotted a bug with Disclaimer back button on the app bar https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues/195 which you may or may not want to fix as part of this PR? I'll leave that up to you as we can do it as a separate fix. But if you have questions about the themes just msg me.

**Pull request merged**